go.mod

```
@@ -16,7 +16,7 @@ require (
 	github.com/prometheus/client_golang v1.14.0
 	github.com/submariner-io/admiral v0.15.0-rc0
 	github.com/submariner-io/shipyard v0.15.0-rc0
-	github.com/submariner-io/submariner v0.15.0-m4.0.20230401111446-1faf6115599a
+	github.com/submariner-io/submariner v0.15.0-m4.0.20230403075149-13e2c1be4465
```

```suggestion
	github.com/submariner-io/submariner v0.15.0-rc0
```

Shouldn't this be just rc0?

---

So it should!